BaseLib/ConfigTree.h

```diff
@@ -0,0 +1,31 @@
+/**
+ * \copyright
+ * Copyright (c) 2012-2015, OpenGeoSys Community (http://www.opengeosys.org)
+ *            Distributed under a Modified BSD License.
+ *              See accompanying file LICENSE.txt or
+ *              http://www.opengeosys.org/project/license
+ *
+ */
+
+#ifndef BASELIB_CONFIGTREE_H_
+#define BASELIB_CONFIGTREE_H_
+
+#include <string>
+#include <boost/property_tree/ptree.hpp>
```

Comment:
1. Unfortunately it is not possible to include ptree_fwd in `ConfigTree.h` because the explicit template declaration requires the full template...
 2. In my opinion no, because `ConfigTree.h` already constains only declarations.